source/changelog/r91.inc

```diff
@@ -0,0 +1,37 @@
+.. _changelog-r91:
+
+*************************
+Release June 22nd 2021
+*************************
+The following changes started rolling out on Tuesday June 22nd 2021.
+The API and backend are being rolled out to all tenants at the same time, and the client will be propagated over the next couple of days.
+
+API and Backend
+===============
+The following new features and improvements are now available in the API and the backend.
+
+* Expanded the summary result information when importing Mastercard Paymentservices OCR files.
+* Adjustments to the BgMax file to handle payments from Autogiro.
+* Added API validation to prevent empty standalone account demands from being generated by billing.
+* Fixed an issue where reminder fees would not propagate correctly to the Invoice Document.
+* Fixed an issue where the reporting subsystem would not update payment agreements on subscriptions when automatically changed.
+* PayEx POPS related payments are now blocked entirely before being sent to PayEx (where they would sometime actually succeed even though the platform is reported shut down).
```

Comment:
This describes an old fix. What we did now was add the duplicate-validation to prevent drawing the same "demand" twice.